demo/index.html

```diff
@@ -27,105 +30,136 @@
     <div class="c-tab c-tab--block c-tab--main">
       <ul class="c-tab__list c-tab__list">
         <li class="c-tab__list__item" tabindex="0">
-          <a href="//zendeskgarden.github.io/" tabindex="-1">Home</a>
+          <a href="/" tabindex="-1">Home</a>
         </li>
         <li class="c-tab__list__item" tabindex="0">
-          <a href="//zendeskgarden.github.io/assets/" tabindex="-1">Assets</a>
```

Comment:
Keep this (and the previous) full URL, otherwise they don't work correctly with the `start:demo` script.

Reply:
Fixed.